packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -405,147 +537,269 @@ export const ForumEventPoll = ({postId, hideViewResults, classes}: {
+                            )}
                           />
                         </LWTooltip>
-                      ))}
-                    </div>
-                  }
-                  placement="bottom"
-                  clickable
-                  className={classes.voteCluster}
+                      </div>
+                    ))}
+                </div>
+              ))}
+            </div>
+            <div
+              className={classes.sliderLine}
+              ref={sliderRef}
+            >
+              {/* Ticks */}
+              <div className={classes.ticksContainer}>
+                {ticks.map((tickIndex) => (
+                  <div
+                    key={tickIndex}
+                    ref={(el) => (tickRefs.current[tickIndex] = el)}
+                    className={classNames(
+                      classes.tick,
+                      isDragging.current && classes.tickDragging,
+                      tickIndex === Math.floor(NUM_TICKS / 2) &&
+                        classes.centralTick
+                    )}
+                  />
+                ))}
+                {/* User Vote */}
+                <div
+                  className={classNames(
+                    classes.userVote,
+                    classes.currentUserVote,
+                    isDragging.current && classes.currentUserVoteDragging,
+                    hasVoted && classes.currentUserVoteActive
+                  )}
+                  onPointerDown={startDragVote}
+                  style={{
+                    left: `${votePos}%`,
+                    transform: "translateX(-50%)",
+                  }}
                 >
-                  <div></div>
-                </HoverOver>
-              })}
-              <div
+                  <LWTooltip
+                    title={
+                      hasVoted ? (
+                        <div className={classes.voteTooltipBody}>
+                          Drag to move
+                        </div>
+                      ) : (
+                        <>
+                          <div className={classes.voteTooltipHeading}>
+                            Click and drag to vote
+                          </div>
+                          <div className={classes.voteTooltipBody}>
+                            Votes are non-anonymous and can be changed at any time
+                          </div>
+                        </>
+                      )
+                    }
+                  >
+                    {currentUser ? (
+                      <UsersProfileImage
+                        user={currentUser}
+                        size={USER_IMAGE_SIZE}
+                        className={classes.userImage}
+                      />
+                    ) : (
+                      <ForumIcon
+                        icon="UserCircle"
+                        className={classes.placeholderUserIcon}
+                      />
```

Comment:
I think this is smaller than the logged in version:

<img width="354" alt="Screenshot 2024-10-02 at 6 52 42 PM" src="https://github.com/user-attachments/assets/81eeae7d-6d8b-4e49-a964-f6175c6b81c1">